Gemfile.lock

```diff
@@ -1035,7 +1035,9 @@ GEM
     chronic (0.10.2)
     climate_control (0.2.0)
     cliver (0.3.2)
-    codecov (0.1.20)
+    cocaine (0.5.8)
```

Comment:
I think the merge broke this -- `cocaine` was renamed to `terrapin`, so I don't think it should be included in `Gemfile.lock`.

Reply:
😭 😭 😭 😭